cmd/gb/generate.go

```diff
@@ -29,6 +29,7 @@ See 'go help generate'.
 	Run: func(ctx *gb.Context, args []string) error {
 		env := cmd.MergeEnv(os.Environ(), map[string]string{
 			"GOPATH": fmt.Sprintf("%s%c%s", ctx.Projectdir(), filepath.ListSeparator, filepath.Join(ctx.Projectdir(), "vendor")),
+			"PATH": fmt.Sprintf("%s%c%s", filepath.Join(ctx.Projectdir(), "bin"), filepath.ListSeparator, os.Environ["PATH"]),
```

Comment:
os.Environ returns a `[]string` not a map, you should try `os.Getenv("PATH")` instead.

Please move the `$PROJECT/bin` to the end of the search path.

Thanks and sorry for the long review delay.